includes/class-simple-local-avatars.php

```diff
@@ -678,4 +716,168 @@ public function get_avatar_rest( $user ) {
 	public function set_avatar_rest( $input, $user ) {
 		$this->assign_new_user_avatar( $input['media_id'], $user->ID );
 	}
+
+	/**
+	 * Migrate the user's avatar data from WP User Avatar/ProfilePress
+	 *
+	 * This function creates a new option in the wp_options table to store the processed user IDs
+	 * so that we can run this command multiple times without processing the same user over and over again.
+	 *
+	 * Credit to Philip John for the Gist
+	 *
+	 * @see https://gist.github.com/philipjohn/822d3521a95481f6ad7e118a7106fbc7
+	 *
+	 * @return int
+	 */
+	public function migrate_from_wp_user_avatar() {
+
+		global $wpdb;
+
+		$count = 0;
+
+		// Support single site and multisite installs.
+		// Use WordPress function if running multisite.
+		// Create generic class if running single site.
+		if ( is_multisite() ) {
+			$sites = get_sites();
+		} else {
+			$site          = new stdClass();
+			$site->blog_id = 1;
+			$sites         = array( $site );
+		}
+
+		// Bail early if we don't find sites.
+		if ( empty( $sites ) ) {
```

Comment:
Are there chances of `$sites` being `empty`?

Reply:
@faisal-alvi I don't believe so but I like to add those as a safe guard before any `foreach` or `while` loops just in case something does go wrong.